src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.foundations.resources :as quo2.resources]
```

Comment:
https://github.com/status-im/status-mobile/tree/develop/src/quo2#avoid-using-quos-version-number-in-namespace-aliases

Reply:
Done